src/main/g8/build.sbt

```diff
@@ -6,6 +6,7 @@ scalaVersion in ThisBuild := "2.13.0"
 
 val macwire = "com.softwaremill.macwire" %% "macros" % "2.3.3" % "provided"
 val scalaTest = "org.scalatest" %% "scalatest" % "3.0.8" % Test
+val scalaXml = "org.scala-lang.modules" %% "scala-xml" % "1.2.0" % Test
```

Comment:
Wait WAT? Why?

Reply:
Trying random things to fix it.  But it isn't a scala-xml thing here.